src/main/java/ru/teti/springcourse/dao/PersonDAO.java

```diff
@@ -1,51 +1,31 @@
 package ru.teti.springcourse.dao;
 
-import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
 import ru.teti.springcourse.models.Person;
 
-import java.sql.*;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.PreparedStatement;
+import java.sql.DriverManager;
 import java.util.ArrayList;
 import java.util.List;
 
-@Component
-public class PersonDAO {
-    private static int PEOPLE_COUNT;
-
-    private static final String URL = "jdbc:postgresql://localhost:5432/first_db";
-    private static final String USERNAME = "postgres";
-    private static final String PASSWORD = "Strong1+7";
-
-    private static Connection connection;
+import static ru.teti.springcourse.dao.DbConstants.*;
 
-    static {
-        try {
-            Class.forName("org.postgresql.Driver");
```

Comment:
I believe there is no need to load class org.postgresql.Driver anymore since java 6.